include/hc_am.hpp

```diff
@@ -19,7 +19,7 @@ struct AmPointerInfo {
     void *      _hostPointer;   ///< Host pointer.  If host access is not allowed, NULL.
     void *      _devicePointer; ///< Device pointer.  
     size_t      _sizeBytes;     ///< Size of allocation.
-    hc::accelerator &_acc;       ///< Device / Accelerator to use.
```

Comment:
As @facao pointed out, hc::accelerator is just a wrapper of KalmarDevice* , and I've verified that sizeof(hc::accelerator) is only 8. So we should be fine copying it around.